app/src/main/res/layout/quest_fire_hydrant_diameter_last_picked_button.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    style="@style/Widget.AppCompat.Button.Small"
```

Comment:
Independently of the rest, I wonder if there is a reason why we do not use material design theme here? In particular, I guess the material-style to choose here would be the "outlined button"

Reply:
I guess there is no reason, I just used the default style that the android layout designer suggested.